### ISSUE TITLE
Add Left, Right, and Full Width Alignment Options to Code Block

### ISSUE THREAD

**Issue by dhruvikpatel18** [WordPress/gutenberg#66960]
## What problem does this address?
The Code Block block currently only offers alignment options for None and Wide, limiting flexibility for users who want more control over the block’s placement in their content. This makes it harder to align the block to the left, right, or full width, similar to other default WordPress blocks.

## What is your proposed solution?
Add additional alignment options (Left, Right, Full Width) to the Code Block block to provide more flexibility in layout customization, ensuring consistency with other default WordPress blocks and improving the overall user experience.

![Image](https://github.com/user-attachments/assets/1a59e073-2d08-489b-b7cd-d67aefae3151)

**Comment by Mamaduka** [comment]
I'm unsure how useful these settings will be for the Code block.

@richtabor, any thoughts since you introduced support for the "Wide" alignment (#50710)?

**Comment by Kallyan01** [comment]
One can easily use layout blocks to do such adjustments if needed , adding separate alignments for code block seems not useful enough !

**Comment by im3dabasia** [comment]
I agree with this https://github.com/WordPress/gutenberg/issues/66960#issuecomment-2474651022, this is something which can be achieved through existing layout blocks.
